packages/protocol/test/liquidity/grandamento.ts

```diff
@@ -260,6 +271,52 @@ contract('GrandaMento', (accounts: string[]) => {
       assertEqualBN(await grandaMento.activeProposalIdsSuperset(1), new BigNumber(2))
     })
 
+    it('fails if the stable token amount would overflow in a unitsToValue call', async () => {
+      // We need to deploy the real StableToken contract because MockStableToken
```

Comment:
Doesn't really matter but I wonder if just modifying `MockStableToken` to remove
```
    uint256 balanceValue = balanceOf(from);
    if (balanceValue < value) {
      return false;
    }
```
 would've been easier rather than having to set up all this scaffolding? Because `balances[from].sub(units);` will revert if it underflows anyways

Reply:
I think in this case since we're testing against the real behavior of StableToken, it might be nice to use the actual contract?

Reply:
Yeah good point, makes sense to me